dialog box.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'dart:ui';
+
+showAlertDialog(BuildContext context) {
+  // set up the button
+  Widget okButton = FlatButton(
+    child: Text("OK"),
+    onPressed: () {
+      return;
+    },
+  );
+
+  // set up the AlertDialog
+  AlertDialog alert = AlertDialog(
+    content: Text(
+      "Player Wins!",
+      textAlign: TextAlign.center,
+      style: TextStyle(
+        color: Colors.orange,
+        fontSize: 25.0,
+      ),
+    ),
+    actions: [
+      okButton,
+    ],
+  );
+
+  // show the dialog
+  showDialog(
+    context: context,
+    builder: (BuildContext context) {
+      return alert;
+    },
+  );
+}
+
+void main() {
```

Comment:
this is not require, this module will be imported into another one,
so render not required